README.md

```diff
@@ -1,2 +1,9 @@
 # BasicProgrammingQuestions
 Interview Ready Basic Programming Questions
+
+### Commits Format
+[branch_name][commit_type]: message
+#### Commit Types:
+    QUE: Question(s) was added
+    UPDT: Update was made in the repo
+    BUG: Bug fix
```

Comment:
you can add/remove types here. or could suggest any other method that you know of